plugins/jabra-gnp/fu-jabra-gnp-device.c

```diff
@@ -576,12 +593,12 @@ fu_jabra_gnp_device_write_image(FuJabraGnpDevice *self,
 	/* progress */
 	fu_progress_set_id(progress, G_STRLOC);
 	fu_progress_add_flag(progress, FU_PROGRESS_FLAG_GUESSED); // FIXME, remove
-	fu_progress_add_step(progress, FWUPD_STATUS_DEVICE_WRITE, 10, "write-partition");
```

Comment:
I think we can loose the fixme too. Isn't there also one other guessed progress steps to fill in too?

Reply:
The other guessed was quite accurate, so I have just removed the guessed flag now. However I'm still getting a "Less than one minute remaining..." console output when executing with fwupdtool, when there is more than one minute remaining. It appears at the start of the writing phase, and I'm not entirely sure how to fix it, or how important it is to fix it.

Reply:
I don't think it's terribly important FWIW. I'd be curious to know if this also happens with fwupd/fwupdmgr too.